release-notes/release-notes55.md

```diff
@@ -41,6 +41,15 @@ The new listbox component for custom dialogs is a composite component with a lab
 
 For information on the listbox dialog component, see: [Dialog components - listbox]({{site.baseurl}}/ui-components/dialogcomponents/#listbox).
 
+### The toolbar drawer can now be controlled using commands
```

Comment:
@tylerkelly13  Looks good. Was just wondering whether 'toggle toolbar' should be capitalized or not below? I have it non-capitalized everywhere else and am not if sure if it needs to consistent or if the capitalization is context dependent.

Also, for `The command allow the Toolbar Drawer...` below, I think `allow` should be `allows`?

Reply:
Did you mean 'toolbar drawer'?

I've added suggestions for you to accept/reject.